pkg/apiserver/logsearch/models.go

```diff
@@ -106,33 +94,31 @@ func (r *SearchLogRequest) Value() (driver.Value, error) {
 }
 
 type SearchTarget struct {
```

Comment:
There is no need to leave this structure :) `RequestTargetNode` already includes everything it needs. The **actual port (10080 / 2379 / ...)** shall be stored in `port`, while the **port (4000 / 2379 / ...)** shall be stored in `DisplayName` (to be used for front-end display only). You can take a look at Profiling to see how it works.